ui-core/src/participant/landing/sections/ParticipationDetailTemplate.tsx

```diff
@@ -14,6 +14,7 @@ import ConfiguredMedia, { MediaConfig, mediaConfigProps, validateMediaConfig } f
 import { TemplateComponentProps } from './templateUtils'
 import { useApiContext } from '../../../participant/ApiProvider'
 import { blurbProp, titleProp } from './SectionProp'
+import { InlineMarkdown, Markdown } from '../../../participant/landing/Markdown'
```

Comment:
cleaning up these imports requires some tsconfig changes. should I do that now or wait until after the vite migration so we can come in and do a full cleanup of ui-core?

Reply:
great question, please don't change tsconfig -- I've been making a ton of tsconfig changes for the vite migration, and so they'll conflict